uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -388,6 +440,21 @@ define([
       delete params.carry_forward_expiration_duration_switch;
     }
 
+    /**
+     * Pre-processes parameters depending on the selected leave category.
```

Comment:
"pre process" implies that the value of each field needs to go through some sort of function, but if i understand this correctly, you are simply assigning default values to the params? If that's the case, please rename the function and variables to better reflect this

Reply:
Done.